common/src/main/java/com/wynntils/screens/OverlayManagementScreen.java

```diff
@@ -0,0 +1,651 @@
+            double newSnapValue;
+            if (entry.getKey().isVerticalLine()) {
+                newSnapValue = entry.getValue() - Math.abs(dragX);
+                dragX = 0;
+            } else {
+                newSnapValue = entry.getValue() - Math.abs(dragY);
+                dragY = 0;
+            }
+
+            if (newSnapValue <= 0) {
+                toBeRemoved.add(entry.getKey());
+                if (entry.getKey().isVerticalLine()) {
+                    dragX = originalX;
+                } else {
+                    dragY = originalY;
+                }
+            } else {
+                edgeAlignmentSnapMap.put(entry.getKey(), newSnapValue);
+            }
+        }
+
+        for (Edge edge : toBeRemoved) {
+            edgeAlignmentSnapMap.remove(edge);
+            alignmentLinesToRender.remove(edge);
+        }
+
+        return new Pair<>(dragX, dragY);
+    }
+
+    private void renderSections(PoseStack poseStack) {
+        for (SectionCoordinates section : OverlayManager.getSections()) {
+            RenderUtils.drawRectBorders(
+                    poseStack, CommonColors.WHITE, section.x1(), section.y1(), section.x2(), section.y2(), 0, 1);
+        }
+    }
+
+    private void renderAlignmentLines(PoseStack poseStack) {
+        for (Map.Entry<Edge, Float> entry : alignmentLinesToRender.entrySet()) {
+            if (entry.getKey().isVerticalLine()) {
+                RenderUtils.drawLine(
+                        poseStack,
+                        CommonColors.ORANGE,
+                        this.width * entry.getValue(),
+                        0,
+                        this.width * entry.getValue(),
+                        this.height,
+                        1,
+                        1);
+            } else {
+                RenderUtils.drawLine(
+                        poseStack,
+                        CommonColors.ORANGE,
+                        0,
+                        this.height * entry.getValue(),
+                        this.width,
+                        this.height * entry.getValue(),
+                        1,
+                        1);
+            }
+        }
+    }
+
+    private void calculateAlignmentLinePositions() {
+        verticalAlignmentLinePositions.clear();
+        horizontalAlignmentLinePositions.clear();
+
+        verticalAlignmentLinePositions.add(0f);
+        horizontalAlignmentLinePositions.add(0f);
+        verticalAlignmentLinePositions.add(1f);
+        horizontalAlignmentLinePositions.add(1f);
+
+        for (int i = 2; i <= ALIGNMENT_LINES_MAX_SECTIONS_PER_AXIS; i++) {
+            for (int j = 1; j < i; j++) {
+                verticalAlignmentLinePositions.add((float) j / i);
+                horizontalAlignmentLinePositions.add((float) j / i);
+            }
+        }
+
+        for (Overlay overlay :
+                OverlayManager.getOverlays().stream().filter(Overlay::isEnabled).collect(Collectors.toSet())) {
```

Comment:
Not that it matters greatly, but a simple `toList()` would be shorter, and in practice it's not relevant that a set would have sufficed. (In practice, Lists are often used in Java even when order is ignored.)